.github/ISSUE_TEMPLATE/monthly-cyhy-environment-redeploy.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Monthly CyHy Environment Redeploy
+about: Monthly redeploy of the CyHy .nvironment
+title: 2020-XX Redeploy CyHy Environment
```

Comment:
Is there a reason you decided against `202X-XX`? You left that in the `Procedure` section, so I think it should be consistent one way or the other.

Reply:
I'm considering pre-loading it for 2021 - what would you think?